src/components/molecules/LiveSchedule/LiveEvent.tsx

```diff
@@ -52,7 +52,9 @@ export const LiveEvent: FC<LiveEventProps> = ({ venue, event }) => {
           {event.description}
         </div>
         <div className="schedule-event-info-room">
-          <div onClick={enterLiveEvent}>{event.room ?? "Enter"}</div>
+          <div onClick={enterLiveEvent}>
+            {event.room ?? "Enter"} - {venue.name}
+          </div>
```

Comment:
[NB, nit, DRY] We could extract this to a common 'dumb' component as it's repeated in `src/components/molecules/EventDisplay/EventDisplay.tsx`